src/components/Courses/Courses.jsx

```diff
@@ -16,6 +18,28 @@ const Courses = () => {
 
   const navigate = useNavigate();
 
+  let courses = useSelector((state) => state.courses.courses);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    axios
+      .get('http://localhost:4000/courses/all')
+      .then(function (response) {
```

Comment:
Please, use AF here:
```suggestion
      .then((response) => {
```

Same case for other usages